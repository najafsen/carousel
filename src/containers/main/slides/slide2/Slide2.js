import React from "react";
import commaRed from "./img/comma-red.svg";
import cake from "./img/cake.svg";
import "./Slide2.scss";

export const Slide2 = () => (
  <div className="slide-content">
    <div className="slide2">
      <img src={commaRed} alt="Comma Red" />
      <h1>
        An experience that makes us feel like we’re covered in maple syrup –
        <i> It’s a good thing</i>
      </h1>
      <img className="bottom-image" src={cake} alt="Cake" />
      <h4>The whole team @</h4>
      <h4>The Pancake Collective™</h4>
    </div>
  </div>
);
