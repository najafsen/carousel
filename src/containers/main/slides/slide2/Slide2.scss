.slide2 {
  display: flex;
  flex: 1;
  flex-direction: column;
  margin: 72px 152px 96px 152px;
  justify-content: center;

  * {
    text-align: center;
  }

  h1 {
    margin-top: 32px;
    font-size: 40px;

    i {
      color: #ff88ad;
    }
  }

  .bottom-image {
    margin-top: 32px;
    margin-bottom: 8px;
  }

  h4 {
    margin: 4px;
  }
}
