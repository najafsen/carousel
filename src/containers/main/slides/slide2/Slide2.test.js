import React from "react";
import { render } from "@testing-library/react";
import { Slide2 } from "./Slide2";

describe("Slide2", () => {
  test("matches snapshot", () => {
    const { asFragment } = render(<Slide2 />);
    expect(asFragment()).toMatchSnapshot();
  });
});
