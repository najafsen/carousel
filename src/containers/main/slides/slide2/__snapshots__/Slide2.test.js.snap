// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide2 matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="slide-content"
  >
    <div
      class="slide2"
    >
      <img
        alt="Comma Red"
        src="comma-red.svg"
      />
      <h1>
        An experience that makes us feel like we’re covered in maple syrup –
        <i>
           It’s a good thing
        </i>
      </h1>
      <img
        alt="Cake"
        class="bottom-image"
        src="cake.svg"
      />
      <h4>
        The whole team @
      </h4>
      <h4>
        The Pancake Collective™
      </h4>
    </div>
  </div>
</DocumentFragment>
`;
